doc/releases/release-notes-3.7.rst

```diff
@@ -63,6 +63,10 @@ Deprecated in this release
  * POSIX API
 
   * Deprecated :c:macro:`PTHREAD_BARRIER_DEFINE` has been removed.
+  * In efforts to use Kconfig options that map directly to the Options and Option Groups in
+    IEEE 1003.1-2017, the following Kconfig options have been deprecated (replaced by):
+
+    * :kconfig:option:`CONFIG_POSIX_LIMITS_RTSIG_MAX` (:kconfig:option:`CONFIG_POSIX_RTSIG_MAX`)
```

Comment:
Suggestion to merge the changes in `doc: release: posix: deprecate CONFIG_POSIX_LIMITS_RTSIG_MAX` to the commit that does the actual deprecation.

Reply:
I wanted to keep release notes to separate commits.

What I could do is squash all of the changes to the release notes to 1 commit at the end.

Reply:
Fixed!